src/utils/Youtube.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+//const API_KEY = "AIzaSyAUl9bRka5hDTPjavDgGwDeBc9Fkqc7_yY";
+// AIzaSyCzZtUxQroQmSDWl_ul_Ym24ADEl_9by1w
+// AIzaSyCNoOErljI06cfv52iwogm4651bBwx
+const KEY = "AIzaSyB7bFSwZazNzpCapLfbxQf3kUBr_n283ss";
```

Comment:
This is sensitive information, you should provide this in configs (.env) files

Reply:
+1